### DISCUSSION TITLE
3.2.3/3.2.4に関する不具合や質問はこちら

### DISCUSSION THREAD

**Discussion by dongaba** [dongaba/TVerRec#120] [Issues]
### 3.2.4 Fixed

- サイトマップとトップページからの動画取得に失敗する問題を修正

## What's Changed
* support episode as keyword by @dongaba in https://github.com/dongaba/TVerRec/pull/113
* Merge from Dev to Beta by @dongaba in https://github.com/dongaba/TVerRec/pull/116
* Merge from dev to beta by @dongaba in https://github.com/dongaba/TVerRec/pull/117
* merge from Dev by @dongaba in https://github.com/dongaba/TVerRec/pull/118
* Fixed sitemap and toppage parse fail issue by @dongaba in https://github.com/dongaba/TVerRec/pull/121
* Fixed sitemap and toppage parse fail issue by @dongaba in https://github.com/dongaba/TVerRec/pull/122

**Full Changelog**: https://github.com/dongaba/TVerRec/compare/v3.2.3...v3.2.4

### 3.2.3 Added

- キーワード指定ファイルに番組エピソードの指定をできる機能を追加
- TVerIDを使っている人のマイページからの番組取得に対応
  - 設定ファイルに$script:myMemberSIDを追加
  - 本機能により、ダウンロード対象番組の管理をブラウザ上のマイページで管理することができるようになります
- Chrome拡張機能のTVerRec AssistantにMember SIDの取得機能を追加(TVerにログインして使用する場合に必要)
  - [使い方はWikiのこちらのページ参照](https://github.com/dongaba/TVerRec/wiki/platform_uid%E3%80%81platform_token%E3%80%81member_sid%E3%81%AE%E5%8F%96%E5%BE%97)
- GUI版の設定画面からMember SIDを指定できる機能を追加
- 個別ダウンロード時に複数URLを指定できる機能を追加

### 3.2.3 Changed

- GUI版の設定画面の高度な設定のグルーピングを変更
- Windows Sandbox環境の環境構築スクリプトを高速化
- Dockerコンテナにvimを同梱

### 3.2.3 Fixed

- 動画移動時に「Cannot create 'example' because a file or directory with the same name already exists.」が出てファイルの移動が出来ない問題を修正
- Windows SandBox環境でmshtaによるメッセージボックス描画が動作しない問題を修正

## What's Changed
* Dev by @dongaba in https://github.com/dongaba/TVerRec/pull/110
* 114 move video error at line143 by @dongaba in https://github.com/dongaba/TVerRec/pull/115
* merge from Dev by @dongaba in https://github.com/dongaba/TVerRec/pull/119

**Full Changelog**: https://github.com/dongaba/TVerRec/compare/v3.2.2...v3.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/dongaba/TVerRec/releases/tag/v3.2.3'>3.2.3</a>.</em>

**Comment by SilentBear555** [comment]
3.2.4で、下記URLが、エラー「URLではありません: True」になります。

バスVS鉄道22 勝てたルートを大公開！負けたチームのリーダーが悔やみまくり！
https://tver.jp/episodes/eplz3bv0i9?p=1155

**Reply by dongaba** [reply]
問題を確認しました、修正します。

**Reply by dongaba** [reply]
3.2.5で修正したつもりです
